pom.xml

```diff
@@ -654,5 +684,9 @@
         <!-- dependency check plugin properties -->
         <dependency.check.maven.version>7.1.1</dependency.check.maven.version>
         <dependency.check.cvss.threshold.value>4</dependency.check.cvss.threshold.value>
+        <!-- -code coverage related properties -->
+        <maven.failsafe.plugin.version>2.15</maven.failsafe.plugin.version>
+        <jacoco.version>0.8.8</jacoco.version>
+        <skip.code.coverage>true</skip.code.coverage>
```

Comment:
By default code coverage is disabled. Since the codecov relevant java agent jar need to be packed in the enforcer. We are avoiding it during release build by using this property.

Reply:
Can't we copy the jar to the the dropins directory and refer from there? It is not good to make the enforcer dependent on this property.